### ISSUE TITLE
void function doom-enlist

### ISSUE THREAD

**Issue by nasyxx** [hlissner/emacs-doom-themes#213]
When I eval (doom-themes-treemacs-config), this error happens.

```elisp
Debugger entered--Lisp error: (void-function doom-enlist)
  doom-enlist(font-lock-string-face)
  doom--treemacs-variable-pitch-labels()
  #f(compiled-function () #<bytecode 0x429bcb41>)()
  eval-after-load(treemacs #f(compiled-function () #<bytecode 0x429bcb41>))
  byte-code("\300\301\302\"\210\303\304!\207" [eval-after-load treemacs #f(compiled-function () #<bytecode 0x429bcb41>) provide doom-themes-treemacs] 3)
  require(doom-themes-treemacs)
  doom-themes-treemacs-config()
```

**Comment by hlissner** [comment]
Sorry about that! `doom-enlist` belongs to [Doom emacs](https://github.com/hlissner/doom-emacs), not this package. It is fixed as of d5dd49b.

**Comment by nasyxx** [comment]
Oh! OK, thank you.
